src/main/java/bestie/task/Event.java

```diff
@@ -0,0 +1,66 @@
+package bestie.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Creates an instance of an event task.
+ */
+public class Event extends Task {
+
+    protected String start;
+    protected String end;
+    private String formattedStartTime;
+    private String formattedEndTime;
+
+    private LocalDateTime parsedStart;
+    private LocalDateTime parsedEnd;
+
+    /**
+     * Creates a new event task.
+     *
+     * @param description Description of the task.
+     * @param start Start time of the task in the format YYYY-MM-DD HHMM.
+     * @param end End time of the task in the format YYYY-MM-DD HHMM.
+     */
+    public Event(String description, String start, String end) {
+        super(description);
+        this.start = start;
+        this.end = end;
+        DateTimeFormatter inputFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+        this.parsedStart = LocalDateTime.parse(start, inputFormat);
+        this.parsedEnd = LocalDateTime.parse(end, inputFormat);
```

Comment:
Code could be separated with a blank line such that one portion is for parsing input and the other portion is for formatting the output for better clarity since the code is long